crates/agglayer-types/src/lib.rs

```diff
@@ -279,6 +292,8 @@ pub struct Certificate {
     /// Aggchain data which is either one ECDSA or Generic proof.
     #[serde(flatten)]
     pub aggchain_data: AggchainData,
+    #[serde(default)]
+    pub custom_chain_data: Vec<u8>,
```

Comment:
This is getting in together with the aggchain proof changes, correct? If not, we'd need a new storage version (or relax the parsing of the existing one).

Reply:
It is indeed only provided with the `aggchain-proof`. I thought the changes that I made in the storage types was enough, do you see something that should be different?

Reply:
> It is indeed only provided with the `aggchain-proof`. I thought the changes that I made in the storage types was enough, do you see something that should be different?

Nope, I think it's good as is.